poetry/masonry/utils/include.py

```diff
@@ -25,7 +26,7 @@ def __init__(
         self._include = str(include)
         self._formats = formats
 
-        self._elements = sorted(list(self._base.glob(str(self._include))))
+        self._elements = sorted(list(self._glob_include()))
```

Comment:
What about

```
self._elements = sorted(list(glob(Path(self._base, self._include), recursive=True)))
```
?

AFAIR `pathlib`'s `glob` doesn't  return the correct case on windows file system. So replacing it in this line with the "normal" glob should be enough, shouldn't it?

Reply:
@finswimmer Your solution returns the "expected string or bytes-like object" error, not sure why...

Reply:
I think that glob() expects the first argument to be string, not Path. If I use str(Path(..., I get another error: 'str' object has no attribute 'is_dir'